### DISCUSSION TITLE
Funding

### DISCUSSION THREAD

**Discussion by FortniteRadio** [SaxyPandaBear/TwitchSongRequests#179] [General]
I’ve looked all over for this functionality - it works and i’m so happy about it. Any place we can donate some funds to the cost of it?

Kras

**Comment by SaxyPandaBear** [comment]
Thanks for reaching out about that. I really appreciate the thought. I'm not super sure what the best way I should handle funding. I thought about making a Patreon but that's probably not the best way. I could make a ko-fi, or you could just PayPal me directly I guess. What would make the most sense to you?

**Comment by FortniteRadio** [comment]
PayPal would be easiest.  Thanks for developing this.  So cool and exactly what i was looking for my stream.

Dan Lewis
________________________________

**Reply by SaxyPandaBear** [reply]
Alright! My paypal is https://paypal.me/SaxyPandaBear?country.x=US&locale.x=en_US 

Absolutely no pressure to send me anything. I'm glad you like the project so much 😁

**Comment by FortniteRadio** [comment]
Sent.

Dan Lewis
________________________________
